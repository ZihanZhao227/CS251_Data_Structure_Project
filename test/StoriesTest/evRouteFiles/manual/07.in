50
0 0 0 CITY Start
1 125 58 CHARGING_STATION CHARGING_STATION_1
2 117 153 RESTAURANT RESTAURANT_2
3 41 122 TOURIST_ATTRACTION TOURIST_ATTRACTION_3
4 125 100 CHARGING_STATION CHARGING_STATION_4
5 187 151 CHARGING_STATION CHARGING_STATION_5
6 4 47 TOURIST_ATTRACTION TOURIST_ATTRACTION_6
7 68 65 REGULAR_WAYPOINT REGULAR_WAYPOINT_7
8 120 38 CHARGING_STATION CHARGING_STATION_8
9 82 127 CITY CITY_9
10 24 142 RESTAURANT RESTAURANT_10
11 170 61 CHARGING_STATION CHARGING_STATION_11
12 129 73 CITY CITY_12
13 198 23 TOURIST_ATTRACTION TOURIST_ATTRACTION_13
14 149 115 CHARGING_STATION CHARGING_STATION_14
15 57 44 CHARGING_STATION CHARGING_STATION_15
16 16 95 CHARGING_STATION CHARGING_STATION_16
17 200 122 CHARGING_STATION CHARGING_STATION_17
18 52 53 RESTAURANT RESTAURANT_18
19 82 127 CHARGING_STATION CHARGING_STATION_19
20 184 161 CHARGING_STATION CHARGING_STATION_20
21 33 91 CITY CITY_21
22 91 93 CITY CITY_22
23 99 110 RESTAURANT RESTAURANT_23
24 18 196 TOURIST_ATTRACTION TOURIST_ATTRACTION_24
25 126 5 TOURIST_ATTRACTION TOURIST_ATTRACTION_25
26 136 88 CHARGING_STATION CHARGING_STATION_26
27 151 173 CHARGING_STATION CHARGING_STATION_27
28 60 116 TOURIST_ATTRACTION TOURIST_ATTRACTION_28
29 119 199 TOURIST_ATTRACTION TOURIST_ATTRACTION_29
30 175 183 TOURIST_ATTRACTION TOURIST_ATTRACTION_30
31 123 132 CHARGING_STATION CHARGING_STATION_31
32 171 102 CHARGING_STATION CHARGING_STATION_32
33 15 62 CHARGING_STATION CHARGING_STATION_33
34 118 63 RESTAURANT RESTAURANT_34
35 35 113 CHARGING_STATION CHARGING_STATION_35
36 120 91 CITY CITY_36
37 14 36 CHARGING_STATION CHARGING_STATION_37
38 125 112 TOURIST_ATTRACTION TOURIST_ATTRACTION_38
39 5 158 CITY CITY_39
40 3 129 RESTAURANT RESTAURANT_40
41 34 141 CHARGING_STATION CHARGING_STATION_41
42 65 142 CHARGING_STATION CHARGING_STATION_42
43 194 193 CHARGING_STATION CHARGING_STATION_43
44 50 186 CHARGING_STATION CHARGING_STATION_44
45 64 183 CHARGING_STATION CHARGING_STATION_45
46 190 83 CHARGING_STATION CHARGING_STATION_46
47 155 183 CHARGING_STATION CHARGING_STATION_47
48 57 29 TOURIST_ATTRACTION TOURIST_ATTRACTION_48
49 200 200 CITY End
