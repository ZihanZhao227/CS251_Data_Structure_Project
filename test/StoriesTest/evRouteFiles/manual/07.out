0 49 80.000 303.236
0 15 1 14 20 49
